docs/AI-PI/avatar_sdk_documentation.md

```diff
@@ -185,3 +185,11 @@ const getGptApiResponse = async (promptRules: string, promptRequest: string) =>
     return data.choices[0].message.content.trim()
 };
 ```
+
+### Using Chat to display messages from conversational mode
+
+```tsx
+import { Chat } from 'alpha-ai-avatar-sdk-react';
```

Comment:
Can you change the import from `'alpha-ai-avatar-sdk-react'` to `'alpha-ai-avatar-sdk-aipi'`?